src/query-client/index.js

```diff
@@ -0,0 +1,6 @@
+import configureQueryClient from './queryClient';
+
+// export * as routines from './routines';
+export default configureQueryClient;
+// export { MUTATION_KEYS } from './config/keys';
```

Comment:
Actually I think there is a way to get mutation keys automatically extracted from `queryClient`, but let me know when you will need them.